ui/app/AppLayouts/Chat/ChatColumn/User.qml

```diff
@@ -11,6 +11,7 @@ Item {
     property string lastSeen: ""
     property string identicon
     property int statusType: Constants.statusType_Online
+    property color offlineColor: Style.current.darkGrey
```

Comment:
Is it working in dark mode, I had to change this to midGrey in other places

Reply:
I did not make it darkGrey, I only made it conditionally transparent.

Reply:
I will check it though in dark mode.

Reply:
Yeah, I know it was like that before, just saw you modified and fix something similar yesterday, so only pinging ;)
Feel free to ignore